src/stores/dialogStore.ts

```diff
@@ -0,0 +1,53 @@
+import {observable, runInAction} from 'mobx';
+import {DialogApi} from '../api/dialogApi';
+import {DialogModel} from '../models';
+import {DialogActionType} from '../models/dialogModel';
+import {RootStore} from './index';
+
+export class DialogStore {
+  @observable pendingDialogs: DialogModel[] = [];
+
+  constructor(readonly rootStore: RootStore, private api: DialogApi) {}
+
+  fetchPendingDialogs = async () => {
+    this.pendingDialogs = [];
+    const response = await this.api.fetchPendingDialogs();
+    runInAction(() => {
+      this.pendingDialogs = response.map(
```

Comment:
Maybe it make sense to create a mapper?

Reply:
Agree, but to be consistent we should cover all models so i'd prefer to do this separately